ledger/accountdb.go

```diff
@@ -71,6 +72,9 @@ var accountsSchema = []string{
 	`CREATE TABLE IF NOT EXISTS accountbase (
 		address blob primary key,
 		data blob)`,
+	`CREATE TABLE IF NOT EXISTS kvstore (
+		key blob primary key,
+		value blob)`,
```

Comment:
@jannotti and other reviewers:  Sanity checking - Do we override `SQLITE_MAX_LENGTH` (for a smaller value) anywhere?  And if so, _can_ the configured value potentially conflict with `kvstore`?

* Docs:  https://www.sqlite.org/limits.html#max_length
* I'm pretty confident the answer is _no_.  But, I raise it anyways because it's my 1st time in the concern's neighborhood.
* Grep-ing the repo I find no overrides.  I also assume that our other `blob` usages are at least as likely to run into the class of problem.

Reply:
We used to encode entire accounts (including all assets and apps) in one blob, which was surely quite large.

Reply:
@jannotti Thanks for the confirmation - resolving.